CONTRIBUTING.md

```diff
@@ -1,35 +1,40 @@
-# Contributing to Direct State Transfer (DST)
+# Contributing to the Hyperledger Lab Perun
 
 Want to contribute? That's great! Any contribution is welcome, e.g.
 documentation, bug reports, feature request, issues, blog posts,
 tutorials, feature implementations, etc. You can contribute code or
 documentation through the standard GitHub pull request model. For large
 contributions we do encourage you to file a ticket in the GitHub issue
-tracking system prior to any development to coordinate with the DST
+tracking system prior to any development to coordinate with the Perun
 development team early in the process. Coordinating up front helps to
 avoid frustration later on.
 
-*Note that this repository only contains the source code of DST. For
-issues / pull requests related only to documentation please refer to the
-[dst-doc](https://github.com/direct-state-transfer/dst-doc) repository.*
+*Note that this repository only contains the source code of the Perun
+Node. For issues / pull requests related only to documentation please
+refer to the
+[perun-doc](https://github.com/direct-state-transfer/perun-doc)
+repository.*
 
 ## Contribution Guideline
 
 * We use the GitHub [issue
-  tracker](https://github.com/direct-state-transfer/dst-go/issues) and
+  tracker](https://github.com/direct-state-transfer/perun-node/issues) and
   this [branching
   model](https://nvie.com/posts/a-successful-git-branching-model/).
 
-* When you want to submit a patch to any DST project, you must [sign a
-  CLA (once) and sign your commits](#sign-the-cla-and-sign-your-work).
+* When you want to submit a patch to any Hyperledger Labs project, you
```

Comment:
Should the scope be limited to "any project in Hyperledger Lab - Perun ?

Reply:
Well, it's actually a requirement for any Hyperledger Labs project IIUC. One could change it to this project though, I guess...

Reply:
You are right that it's a requirement for any Hyperledger Labs project.

But, I was thinking of the limited scope because my view was that this Contribution Guideline is presented in the context of `perun-node` or Hyperledger Lab -Perun, that `perun-node` is a part of.